doc/cascadia/profiles.schema.json

```diff
@@ -717,6 +724,11 @@
           ],
           "type": "string"
         },
+        "bellStyle": {
+          "default": "audible",
+          "description": "Controls what happens when the application emits a BEL character. When set to \"audible\", the Terminal will play a sound. When set to \"none\", nothing will happen.",
+          "$ref": "#/definitions/BellStyle"
```

Comment:
i'm glad it caught this b/c it wouldn't have been a valid schema otherwise!@ :wink: